CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.4.0
+
+This is a feature and maintenance release.
+
+* [TK-247](https://tickets.puppetlabs.com/TK-247) Added tests for Path
+  Traversal Attacks.
+* Add `add-websockets-handler` to WebserverService and WebroutingService
```

Comment:
Could we maybe be a little more verbose here, and say something like "add experimental support for websockets, via new `add-websockets-handler` blah blah, and corresponding client protocol `WebSocketProtocol`"?

Also did we document this as experimental in the README?